### PR TITLE
feat(ICKY-767): Contentful cms integration

### PR DESCRIPTION
**Pull request by kibo-nehabhanushali**
feat(ICKY-767): Contentful cms integration

### CODE REVIEW

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
Move the "components/PreviewMode" to the "cms" folder and delete the duplicated files from "components/cms/home". They are already in the "cms/components" folder

**Comment by kibo-nehabhanushali** [comment]
> Move the "components/PreviewMode" to the "cms" folder and delete the duplicated files from "components/cms/home". They are already in the "cms/components" folder

Done Resolved

**Review by TriptiVermaXP — APPROVED** [review]

**Review by kibo-geetanshuchhabra — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,15 +61,26 @@ CONTENTFUL_SPACE_ID=12345_Secret_SpaceId
...
+- `clientId` - Unique Application (Client) ID of your Application.
+- `sharedSecret` - Secret API key used to authenticate application. Viewable from your [Kibo eCommerce Dev Center](https://mozu.com/login).
 
 Visit [Kibo documentation](https://apidocs.kibong-perf.com/?spec=graphql#auth) for more details on API authentication
 
+next.config.js example
+
+```bash
+contentFulAccessToken= 123_access_token
+contentFulPreviewToken= 123_preview_token
```
Comment (kibo-kevinwatts) [inline README.md]: whenever you're adding examples to docs, use the same format. "Next.config.js" is a javascript file
Dont use bash lines.

I'm not sure what these values mean. They don't map to environment variables or anything in the config

Add the relevant snippet from the next config here in the docs
and, in one line, explain what/why they're here. 
Mention,

```javascript
// next.config.js
{
  //...other config
   contentful: {
      accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
      previewAccessToken: process.env.CONTENTFUL_PREVIEW_ACCESS_TOKEN,
      URL: `https://graphql.contentful.com/content/v1/spaces/${process.env.CONTENTFUL_SPACE_ID}/environments/master`,
    }
  }
```

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kibo-nehabhanushali) [reply README.md]: Resolved , updated readme as mentioned

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**